telemetry_api/lib/telemetry_api_web/controllers/operator_controller.ex

```diff
@@ -11,8 +12,8 @@ defmodule TelemetryApiWeb.OperatorController do
     render(conn, :index, operators: operators)
   end
 
-  def create_or_update(conn, operator_params) do
-    with {:ok, %Operator{} = operator} <- Operators.update_operator_version(operator_params) do
+  def create_or_update(conn, %{"version" => version, "signature" => signature} = attrs) do
+    with {:ok, %Operator{} = operator} <- Operators.update_operator(version, signature, attrs) do
```

Comment:
Maybe we can send the `attrs` directly here:
```Elixir
Operators.update_operator(attrs)
```

Reply:
We could, but since both `version` and `signature` are required arguments I like it better to have them explicitly as parameters. I know we could do something like:
`def update_operator(%{"version" => version, "signature" => signature} = attrs)` 
 but I prefer it the other way.